weather.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Plz, check the comments from this MR

https://github.com/Kross55/YakushenkoFrontEndPro29.01.23/pull/11